src/components/ClockFace.jsx

```diff
@@ -1,20 +1,26 @@
-import React, { useState, useEffect, createContext } from "react";
-import classnames from "classnames";
-import SettingsModal from "./SettingsModal";
-import "./ClockFace.scss";
+import React, { useState, useEffect, createContext } from 'react';
+import PropTypes from 'prop-types';
+import classnames from 'classnames';
+import { CircularProgressBar } from './CircularProgressBar';
+import './ClockFace.scss';
 
 export const ModalContext = createContext();
 
-const ClockFace = () => {
+const ClockFace = (props) => {
   const [sessions, setSessions] = useState([1500, 300, 1500, 300]);
   const [isModalOpen, setIsModalOpen] = useState(false);
   const [isActive, setIsActive] = useState(false);
   const [currentSessionIndex, setCurrentSessionIndex] = useState(0);
   const [currentSeconds, setCurrentSeconds] = useState(sessions[0]);
 
+  const [totalSeconds, setTotalSeconds] = useState(currentSeconds);
+  const timeProgress = (currentSeconds / totalSeconds) * 100;
```

Comment:
TotalSeconds is the total number of seconds for each session. I use this to calculate the % for the progress bar